.github/workflows/default-branch-cache.yaml

```
@@ -2,6 +2,7 @@
 name: default-branch-cache
 
 on:
+  workflow_dispatch:
```

Appears incorrect. See https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch
```suggestion
on workflow_dispatch:
```

nit: please link docs on these kinds of changes in the commit comments somewhere

---

>please link docs

Noted, updated the description. 

Your suggestion is incorrect though. I edited the file in the browser and there were no complaints

---

https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#using-activity-types-and-filters-with-multiple-events